CMakeExt/CompilerFlags.cmake

```diff
@@ -170,9 +170,9 @@ set(CMAKE_CXX_FLAGS_RELEASE
     "${CMAKE_CXX_FLAGS_RELEASE} ${CXX_ENV_SETUP_FLAGS}")
 
 set(CMAKE_C_FLAGS_DEBUG
-    "${CMAKE_C_FLAGS_DEBUG} ${CC_STD_FLAG} ${CXX_OMP_FLAG} ${CC_REPORT_FLAG} ${CC_WARN_FLAG} -O0 -DDASH_DEBUG ${CC_GDB_FLAG}")
+    "${CMAKE_C_FLAGS_DEBUG} ${CC_STD_FLAG} ${CXX_OMP_FLAG} ${CC_REPORT_FLAG} ${CC_WARN_FLAG} -Ofast -DDASH_DEBUG ${CC_GDB_FLAG}")
```

Comment:
Exactly that is my point, seeing the `<optimized out>` annotation on a variable that I'm trying to print because I want to know it's value drives me nuts and requires me to reconfigure, rebuild, and rerun. My understanding is that this likely won't happen with `-Og`, although I do not have empirical evidence at hand atm. After all, it's meant to be a debug build in general, not a build to debug aliasing-related issues.